packages/passport/sdk/src/Passport.ts

```diff
@@ -299,4 +297,31 @@ export class Passport {
     const getUserInfoResult = await this.multiRollupApiClients.passportProfileApi.getUserInfo({ headers });
     return getUserInfoResult.data.linked_addresses;
   }
+
+  public async linkExternalWallet(
+    type: string,
+    walletAddress: string,
+    signature: string,
+    nonce: string,
+  ): Promise<LinkedWallet> {
```

Comment:
Can we update this function to accept a single object as an argument, rather than 4 different arguments please?